docs/development.md

```diff
@@ -0,0 +1,38 @@
+# Setting Up a Development Environment
+
+
+## Table of contents
+
+1. [Automatically format code and check for code styling](#python-styling)
+
+## Automatically format code and check for code styling <a name='python-styling'></a>
+
+Presidio services are PEP8 compliant and continuously enforced on style guide issues during the build process using `flake8`.
+
+Running flake8 locally, using `pipenv run flake8`, you can check for those issues prior to committing a change.
+
+To make stuff easier, you can use pre-commit hooks to verify and automatically format code upon a git commit, using `black`:
+
+1. [Install pre-commit package manager locally.](https://pre-commit.com/#install) 
+
+1. From the project's root, enable pre-commit, installing git hooks in the `.git/` directory:
+
+    `pre-commit install`
+
+2. Commit non PEP8 compliant code will cause commit failure and automatically format your code using `black`, as well as checking code formatting using `flake8` 
```

Comment:
Worth mentioning that although we are PEP8 compliant, the max line width is 88?

Reply:
seems like noise here, maybe above when I state 'Presidio services are PEP8 compliant..' ?
Maybe 'Presidio services are PEP8 compliant (with minor exclusions)'

Reply:
I agree. let's just keep it as PEP8 compliant.